worker/pkg/workflows/datasync/activities/activities.go

```diff
@@ -620,6 +640,14 @@ func parseJavascriptForColumnName(jsCode, targetWord, replacementWord string) st
 	return strings.ReplaceAll(jsCode, targetWord, replacementWord)
 }
 
+func constructJavascriptCode(jsCode, col string) string {
+	if jsCode != "" {
+		return fmt.Sprintf(`function fn1(value){%s};const input = benthos.v0_msg_as_structured();const output = { ...input };output["%[2]s"] = fn1(input["%[2]s"]);benthos.v0_msg_set_structured(output)`, jsCode, col)
```

Comment:
Please use newlines here. this is not very readable.

Reply:
Also this still isn't quite right. You need to take all of the columns that are using a js processor and build a single script that processes all of them at once. We dont want to split that across N processors. Minimize benthos overhead by having a single processor for handling javascript.

Reply:
it's producing this as an output:

`processors:
    - javascript:
        code: |-
            (() => {
            function fnname(value) {
                var payload = value+=" hello";return payload;
            };
            const input = benthos.v0_msg_as_structured();
            const output = { ...input };
            output["name"] = fnname(input["name"]);
            benthos.v0_msg_set_structured(output);


            function fnfirst_name(value) {
                var payload = value+=" firstname";return payload;
            };
            const input = benthos.v0_msg_as_structured();
            const output = { ...input };
            output["first_name"] = fnfirst_name(input["first_name"]);
            benthos.v0_msg_set_structured(output);
            })();`

where both transformers for different columns (name & first_name) are in the same javascript processor.